src/grafana_dashboards/MongoDB_ReplSet_Summary.json

```diff
@@ -468,7 +468,7 @@
             },
             "gridPos": {
                 "h": 8,
-                "w": 24,
```

Comment:
needed?

Reply:
Might not be needed indeed.
I will spin everything up again so I can check